lib/tasks/osc.rake

```diff
@@ -74,6 +74,26 @@ namespace :osc do
     end
   end
 
+  def version_changed? updated_spec_file
+    begin
+      file = Tempfile.new('yast-rake')
+      file.close
+      sh "osc -A '#{obs_api}' cat '#{obs_sr_project}' '#{package_name}' '#{package_name}.spec' > #{file.path}"
```

Comment:
Good point. I have tested and version_changed? is returning true if the package does not exist in OBS. So it will be packaged.